### DISCUSSION TITLE
how to add a table when the storage has been existed?

### DISCUSSION THREAD

**Discussion by SwordJoe** [fnc12/sqlite_orm#1235] [Q&A]
hello, sorry to disturb you again. 
I use sqlite_orm::make_storage() to create a storage, how should i do to add a table into this storage?  It seems that i don't find a function to create a table like this way: `myStorage.create_table()`.

**Comment by fnc12** [comment]
hello. One cannot do it cause the schema is static. If we made the schema dynamic fancy template API will be unavailable. That's why the schema has to be static. I am sure your case can be covered with a static schema

**Reply by SwordJoe** [reply]
ok, i know it now , thanks
